project-fortis-pipeline/docs/user/admin.md

```diff
@@ -0,0 +1,88 @@
+# Admin Settings
+
+Project Fortis comes with an admin page with which you can edit site configurations and also provide details on how events should be monitored and processed. Through the admin site, you will be able to configure site settings, blacklist terms, whitelist terms (watchlist), streams (twitter, facebook, etc.), monitored places, and trusted sources.
+
+Once you deployed a site with `deploy to azure`, you may view the admin site by navigating to: http://localhost:8888/#/site/{YOUR_SITE_NAME}/admin
+
+## Site Settings
+
+Manage service tokens and set up how the site should be displayed.
+
+| Value               | Description          |
+| ------------------- | ------------- |
+| Site Name           | Used as part of the admin url.  |
+| Site Title          | Displayed on the navigation bar.|
+| Bounding Box        | By default follows `Who's on First (WOF)`. The format of the WOF bounding box is: `maxY,minX,minY,maxX`. `Divipola` is also supported.                           |
+| Header Logo Banner  | Image path of banner.  |
+| Supported Languages | A comma delimited string with languages formatted as language codes like `en` or `ar`. The admin site can translate to any of the supported languages.
+| Default Language    | The admin site will be displayed in this language. |
+| Feature Service     | The feature service connection string formatted as `postgres://frontend:{your_password_here}@127.0.0.1/features`. More documentation on the feature service [here](https://github.com/CatalystCode/featureService).  |
```

Comment:
The feature service database connection string is not configurable. Reading the implementation https://github.com/CatalystCode/project-fortis-mono/blob/7ef5d50850c0e49bb569d12b45dc30967c62c3f7/project-fortis-interfaces/src/components/Admin/AdminSettings.js#L180-L184 informs us that this setting maps to the feature service namespace so it should be `wof` (for whole-world places based on open street maps) or `divipola` (for detailed Colombia places based on the Colombian government data-set).